.github/actions/setup_bazel_nix/action.yml

```diff
@@ -1,5 +1,5 @@
-name: Setup bazel
-description: Setup Bazel for CI builds and tests
+name: Setup bazel and Nix
```

Comment:
Can we keep the cachix step and comment it out? I guess we will need it from time to time.